JavaComparator/Comparator.java

```diff
@@ -0,0 +1,133 @@
+import java.util.ArrayList;
+
+public class Comparator {
+    String file_orig, file_tripped;
+    String file_extension;
+
+    Comparator(String Path1, String Path2) {
```

Comment:
Change the argument names to reflect what they mean. Example originalFilePath, roundtrippedFilePath